public/app/features/snapshot/snapshot_ctrl.ts

```diff
@@ -0,0 +1,43 @@
+///<reference path="../../headers/common.d.ts" />
+
+import angular from 'angular';
+import _ from 'lodash';
+
+export class SnapshotsCtrl {
+
+  /** @ngInject */
+  constructor(backendSrv, $scope) {
+    $scope.init = function() {
```

Comment:
don't use scope, look again at https://github.com/grafana/grafana/blob/master/public/app/features/apps/edit_ctrl.ts 

You will need to update the html partial to bind the controller